engine/server/sv_save.c

```diff
@@ -2433,10 +2433,9 @@ int GAME_EXPORT SV_GetSaveComment( const char *savename, char *comment )
 
 		// split comment to sections
 		if( Q_strstr( savename, "quick" ))
-			Q_strncat( comment, "[quick]", CS_SIZE );
+			Q_snprintf( comment, sizeof( comment ), "[quick]%s", description );
 		else if( Q_strstr( savename, "autosave" ))
-			Q_strncat( comment, "[autosave]", CS_SIZE );
-		Q_strncat( comment, description, CS_SIZE );
+			Q_snprintf( comment, sizeof( comment ), "[autosave]%s", description );
```

Comment:
It seems incorrect. `description` should be in saved no matter if `savename` is not quick or autosave.

The correct strcat-less replacement would be: 
```
		if( Q_strstr( savename, "quick" ))
			Q_snprintf( comment, sizeof( comment ), "[quick]%s", description );
		else if( Q_strstr( savename, "autosave" ))
			Q_snprintf( comment, sizeof( comment ), "[autosave]%s", description );
		else Q_strncpy( comment, description, sizeof( comment ));
```